src/components/slide/slide.js

```diff
@@ -293,10 +303,17 @@ export default function Slide({
             <AnimatedDiv
               ref={setStepContainer}
               onClick={handleClick}
-              tabIndex={
-                Object.entries(frameOverrideStyle).length > 0 ? 0 : undefined
-              }
-              style={{ ...springFrameStyle, ...frameOverrideStyle }}
+              tabIndex={inOverviewMode && isActive ? 0 : undefined}
+              style={{
+                ...springFrameStyle,
+                ...frameOverrideStyle,
+                ...(inOverviewMode &&
+                  hover && {
+                    outline: '2px solid white'
```

Comment:
created `isOverviewMode` boolean to fix small bug where the hover styling was still being applied when a user was in DEFAULT_MODE:

![spectacle-hover-fix](https://user-images.githubusercontent.com/16032631/101220246-3596a400-363a-11eb-8114-719dd7c5aa55.gif)